packages/jupyter-ai/src/components/chat.tsx

```diff
@@ -216,9 +218,20 @@ export function Chat(props: ChatProps): JSX.Element {
                   <Box />
                 )}
                 {view === ChatView.Chat ? (
-                  <IconButton onClick={() => setView(ChatView.Settings)}>
-                    <SettingsIcon />
-                  </IconButton>
+                  <Box sx={{ display: 'flex' }}>
+                    <Tooltip title="New Chat">
```

Comment:
Also, can we just hide the "New chat" button if the user is on the welcome screen? You should be able to do this by wrapping the element in a conditional expression:

```ts
{!showWelcomeMessage && (
  ...
)
```

Reply:
This will require re-rendering the snapshots; you should be able to trigger it yourself by posting `please update snapshots` as a comment on this PR, but if you're not able to, our team can help with this. 👍